socorro/external/es/supersearch.py

```diff
@@ -214,6 +222,8 @@ def get(self, **kwargs):
                         results_number = param.value[0]
                     elif param.name == '_facets_size':
                         facets_size = param.value[0]
+                    elif param.name == '_histogram_interval.date':
+                        histogram_interval_date = param.value[0]
```

Comment:
I'm sure it's correct but why the first one? Perhaps explain it to a dummy like me with a code comment.